client/src/partials/inventory/configuration/units/modals/actions.modal.js

```diff
@@ -34,19 +32,13 @@ function InventoryUnitActionsModalController(InventoryUnit, Notify, Instance, Da
   /** add inventory unit */
   function addUnit(record) {
     return InventoryUnit.create(record)
-    .then(function (res) {
-      return res;
-    })
-    .catch(Notify.errorHandler);
+      .catch(Notify.errorHandler);
```

Comment:
I don't think that `Notify.errorHandler` exists. I know this is a re-factor but it would be good to clean these up now - this error will be thrown as a client error silently. 

🐛